### PR TITLE
Jessie arcade

### PR DESCRIPTION
**Pull request by MIhnatco**
Adding 'Jessie' to map arcade

### CODE REVIEW

**Review by r4pt0s — CHANGES_REQUESTED** [review]
@MIhnatco please check the failed checks output.
As far I saw, you have used a method which doesn't exist anymore because it got replaced with another one

**Comment by r4pt0s** [comment]
@MIhnatco and could you maybe add at least a different conversation text?
Otherwise Jesse would just be a copy of Mister Fu :sweat_smile:

File: src/interactions/map_start/misterFu.interaction.js
Comment (r4pt0s) [inline src/interactions/map_start/misterFu.interaction.js:1]: Please revert the changes to how this file looks now
https://github.com/zero-to-mastery/ZTM-Quest/blob/main/src/interactions/map_start/misterFu.interaction.js

**Review by r4pt0s — CHANGES_REQUESTED** [review]

File: src/interactions/map_arcade/jessie.interaction.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import { npcInteractionHandler } from '../handler.interactions';
+
+import { conversationJessie, jessie } from '../../constants';
+
+export const interactionWithJessie = (player, k, map) => {
+    player.onCollide('jessie', () => {
+        player.isInDialog = true;
+
+        npcInteractionHandler(player, jessie.name, k, () => {
```
Comment (b-hexsoul) [inline src/interactions/map_arcade/jessie.interaction.js:11]: ```npcInteractionHandler``` is being used incorrectly in this exported function. Please look at bruno.interaction.js

**Review by b-hexsoul — CHANGES_REQUESTED** [review]

File: src/gameObjects/map_arcade/jessie.gameObject.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { scaleFactor } from '../../constants';
+
+export const jessie = (k, map, spawnpoints) => {
+    k.loadSprite('jessie', './characters.png', {
```
Comment (r4pt0s) [inline src/gameObjects/map_arcade/jessie.gameObject.js:4]: the path to the assets has changed it is ow `./assets/sprites/characters.png`
Can you fix this please :pray: ?

**Review by r4pt0s — CHANGES_REQUESTED** [review]
@MIhnatco could you plase make the requested changes?
Thank you!

**Review by r4pt0s — APPROVED** [review]

**Pull request merged**